change_the_subject.gemspec

```diff
@@ -26,9 +26,11 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  # Uncomment to register a new dependency of your gem
-  # spec.add_dependency "example-gem", "~> 1.0"
+  spec.add_dependency "yaml"
 
-  # For more information and examples about making a new gem, checkout our
-  # guide at: https://bundler.io/guides/creating_gem.html
+  spec.add_development_dependency "pry-byebug"
```

Comment:
If `lib/change_the_subject.rb` has `require "pry-byebug"`, if the application doesn't have pry-byebug already, won't it error, if here it's only in as a development dependency? (bibdata has it as a development & test dependency)

Reply:
Yep. Good catch.

Reply:
Fixed.